python-checks/src/test/resources/checks/nonCallableCalled.py

```diff
@@ -269,9 +269,9 @@ def foo():
 def using_isinstance_with_runtime_type():
     my_non_callable = MyNonCallable()
     if isinstance(my_non_callable, whatever):
-        my_non_callable() # Noncompliant
+        my_non_callable()
```

Comment:
This was not a FP, because `my_non_callable` does not come from a type hint (it is `EXACT` source).
Our previous implementation only removed type information for declared types, not for runtime type. I think we should keep it that way.

If it's not possible I'd expect at least a ticket to fix this now FN.